http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/package.scala

```diff
@@ -45,6 +52,17 @@ package object api {
         case None    => d2
       }
   }
+
+  def extractAppContext(span: Option[Span]): Directive1[ApplicativeAsk[IO, AppContext]] =
+    //val traceIdFromHeader = req.headers.get(CaseInsensitiveString("X-Cloud-Trace-Context")).map(x => TraceId(x.value))
```

Comment:
can you remove the commented out code

Reply:
Actually just did here: https://github.com/DataBiosphere/leonardo/pull/1685